src/components/panels/WindowContainer.tsx

```diff
@@ -97,18 +102,19 @@ export const MainPanel: FunctionComponent<MainPanelProps> = (props: MainPanelPro
                 />
                 <ItemsContainer>
                 {
-                    props.tabData.map((tab, index) => {
+                    props.panelData.map((panel, index) => {
                         return (
-                            <TabButtonComponent
-                                id={tab.id ? tab.id : tab.title.slice(0, 1)}
-                                idColour={tab.idColour}
-                                tabColour={tab.tabColour}
+                            <PanelIconComponent
+                                id={panel.id ? panel.id : panel.title.slice(0, 1)}
```

Comment:
Is it safe to use only the first character of title if `panel.id` is not present? Wouldn't be better to convert title into an id-friendly format like `'My Title'` to `'my-title'` and set that as id instead?

Reply:
I may need to think of a better name for panel.id, it is being used here as the text inside the panel icon, the full title would be too long for this